org-ac.el

```diff
@@ -236,5 +243,20 @@
   (add-hook 'org-mode-hook 'org-ac/setup-current-buffer t))
 
 
+(defun org-ac/file-candidate ()
```

Comment:
I think the following is better from the definition of `ac-source-org-ac-file`.

```lisp
(defun org-ac/file-candidate ()
  "Adds [file: to the normal file completition, plus allows relative paths"
  (if (string-match "^[~./]+" ac-prefix)
      (ac-filename-candidate)
    (let ((ac-prefix (concat "./" ac-prefix)))
      (mapcar (lambda (path) (substring path 2))
              (ac-filename-candidate)))))
```

`ac-filename-candidate` can't return the nodes of current directory if `ac-prefix` is not a path format as you know.
Therefore, only if so, make `ac-prefix` be localize and update it.
 And, simple logic is better to repair the modified return value of `ac-filename-candidate` for low cost.